source/common/config/well_known_names.cc

```diff
@@ -1,8 +1,37 @@
 #include "common/config/well_known_names.h"
 
+#include "absl/strings/str_replace.h"
+
 namespace Envoy {
 namespace Config {
 
+namespace {
+
+const std::string& wordRegex() { CONSTRUCT_ON_FIRST_USE(std::string, R"([^\.]+)"); }
+
+std::string expandRegex(const std::string& regex) {
+  return absl::StrReplaceAll(
+      regex, {{"<ADDRESS>",
+               R"((?:\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}_\d+|\[[_aAbBcCdDeEfF[:digit:]]+\]_\d+))"},
+              {"<CALLSITE>", wordRegex()},
```

Comment:
This looks great. But WDYT of collapsing all these wordRegex() substitution into one generic `<NAME>`? That would leave you with about 4 subtitutions.

Reply:
Yeap, that's more readable. Updated.